container-storage-setup.sh

```diff
@@ -1230,9 +1237,10 @@ if [ "${_STORAGE_OUT_FILE}" = "/etc/sysconfig/docker-storage" ]; then
 fi
 
 # If user has overridden any settings in $_STORAGE_IN_FILE
-# take that into account.
+# take that into account and make a copy for rollback.
```

Comment:
This is not a rollback file, it is a current state file?  I think we should call it something else.  Also should we only make the backup when we create the _STORAGE_OUT_FILE, this way we could rollback on a failure.